common/src/main/java/org/apache/celeborn/common/network/TransportContext.java

```diff
@@ -209,17 +209,27 @@ public TransportChannelHandler initializePipeline(
 
   private SSLFactory createSslFactory() {
     if (conf.sslEnabled()) {
+
```

Comment:
the check on line 213 confused me a little and I had to read the code a little to be sure this would work. Could we refactor this slightly?

IMO there should be 2 separate blocks for readability:

1. "if SSL is enabled and auto SSL enabled"
2. "if SSL is enabled and auto SSL is not enabled"

The bundling of the keys are valid check is confusing. It works today but as the code evolves, if we update that function we might break here.

In addition, the error message if keys are not present here could probably use some improvement (though outside the scope of this PR potentially). Since in this case we only care about keystore being present right now, we can be more specific.

Reply:
Given how the code has evolved, we dont even need to special case `!autoSslEnabled` here.
Earlier, I was creating the self-signed certificate at this place; but now that it has been moved to `SSLFactory`, this is not required.

Thanks for pointing this out !

Reply:
Updated the message as well, thanks !